integration-tests/model/test_time_discretisation.py

```diff
@@ -8,8 +8,9 @@ def run(timestepper, tmax, f_end):
     return norm(timestepper.fields("f") - f_end) / norm(f_end)
 
 
-@pytest.mark.parametrize("scheme", ["ssprk", "implicit_midpoint",
-                                    "RK4", "Heun", "BDF2", "TR_BDF2", "AdamsBashforth", "Leapfrog", "AdamsMoulton"])
+@pytest.mark.parametrize("scheme", ["ssprk", "TrapeziumRule",
+                                    "RK4", "Heun", "BDF2", "TR_BDF2", "AdamsBashforth",
```

Comment:
Should we add some of the IMEX schemes here to test them?

Reply:
The issue here is the test is a prescribed transport test, hence no option for implicit and explicit terms.. Could change or add another test?

Reply:
We discussed offline: adding a new `test_imex.py` which could test these on something simple like a wave equation with an analytic solution

Reply:
I've added test_IMEX.py. It uses a continuity form and splits the term to treat q*div(u) implicitly and u.grad(q) explicitly.